### PR TITLE
Update documentation for Dockerfile, add location of tbevents, fix link

### PR DESCRIPTION
**Pull request by nvkevlu**
Update documentation for Dockerfile, add location of tbevents, fix link.

Adds location of tbevents in hello example, fix link for better navigation in documentation. Updates the Dockerfile to be more consistent, explains the different Dockerfile with comment.

### Description

A few sentences describing the changes proposed in this pull request.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# For Running NVIDIA FLARE in a Docker container, see
+# https://nvflare.readthedocs.io/en/main/quickstart.html#containerized-deployment-with-docker
+# This Dockerfile is primarily for building Docker images to publish for dashboard.
+
 FROM python:3.8
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install zip
```
Comment (YuanTingHsieh) [inline docker/Dockerfile:6]: what is the purpose of this docker, to build a dashboard image?
Should we use FROM python:3.12?

Comment (nvkevlu) [reply docker/Dockerfile:6]: @IsaacYangSLA said that this was for building the docker images to push to dockerhub for the dashboard, I am not sure if we should update the python version now or at a future time

Comment (YuanTingHsieh) [reply docker/Dockerfile:6]: ok we can also do this in a different PR. @IsaacYangSLA ^

**Review by YuanTingHsieh — APPROVED** [review]
LGTM

**Comment by YuanTingHsieh** [comment]
@nvkevlu do we want the changes to be in 2.5 branch as well?

**Comment by nvkevlu** [comment]
Yes, I can create a PR for that branch

**Pull request merged**